### PR TITLE
Refactor/usedialogutil

### PR DESCRIPTION
**Pull request by kwendel**
Every class now uses the dialogutil for pop-ups.

### CODE REVIEW

File: src/main/java/tudelft/ti2806/pl3/ui/util/DialogUtil.java [hunk 1]
```diff
@@ -26,4 +28,51 @@ public static boolean confirm(String title, String message) {
...
+	 * @param view
+	 * 		to display
+	 * @param title
+	 * 		of the pop-up
+	 */
+	public static void displayQuestionMessageWithView(Component view, String title) {
+		JOptionPane.showMessageDialog(null, view, title, JOptionPane.QUESTION_MESSAGE);
+	}
+
+	/**
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/ui/util/DialogUtil.java]: Needs a dot after the sentence

**Comment by TomBrouws** [comment]
Looks good apart from the dot!

**Comment by borismattijssen** [comment]
Cool! Looks good!

**Comment by mathieupost** [comment]
Looks good! Merge!

**Comment by Delfil** [comment]
Looks good

**Pull request merged**